Assets/Player.cs

```diff
@@ -58,6 +62,8 @@ public void Update()
             MoveDown();
 
         }
+
+        EndGame();
```

Comment:
Raczej nazwę bym zmienił. Może na `TryEndGame()` lub coś podobnego, bo ta nazwa sugeruje, że gra zakończy się przy każdym wywołaniu `Update()`.

Reply:
Change to TryEndGame